hathor/utils/pydantic.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel as PydanticBaseModel, Extra
+from pydantic.generics import GenericModel as PydanticGenericModel
+
+
+class BaseModel(PydanticBaseModel):
```

Comment:
Missing docstring explaining what it is. Maybe add some guidance for whoever is gonna use it in the future.

Reply:
Done!

Reply:
Thanks!